spec/requests/stripe_model_callbacks/subscription_schedule/canceled_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "rails_helper"
+
+describe "subscription_schedule created" do
+  describe "#execute!" do
+    subject(:stripe_subscription_schedule) { StripeSubscriptionSchedule.last }
+
+    let(:stripe_event) { mock_stripe_event("subscription_schedule.canceled") }
+    let(:stripe_id) { "sub_sched_1GhwDXJ3a8kmO8fmxeer3WCy" }
+    let(:stripe_subscription_schedule_phase_plan) { stripe_subscription_schedule_phase.stripe_subscription_schedule_phase_plans.first }
+    let(:stripe_subscription_schedule_phase) { stripe_subscription_schedule.stripe_subscription_schedule_phases.first }
+
+    it "responses with 200" do
+      stripe_event
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "creates a subscription_schedule" do
+      expect { stripe_event }
+        .to change(StripeSubscriptionSchedule, :count).by(1)
+    end
+
+    it "saves data from stripe", :aggregate_failures do
+      stripe_event
+
+      expect(stripe_subscription_schedule.stripe_id).to eq stripe_id
+      expect(stripe_subscription_schedule.billing).to eq "charge_automatically"
+      expect(stripe_subscription_schedule.billing_thresholds_amount_gte).to eq nil
+      expect(stripe_subscription_schedule.billing_thresholds_reset_billing_cycle_anchor).to eq nil
```

Comment:
Lets use `have_attributes` in the future :-) Its  much more readable because a lot less code is used :-) 
```suggestion
      expect(stripe_subscription_schedule).to have_attributes(
        stripe_id: stripe_id,
        billing: "charge_automatically",
        billing_thresholds_amount_gte: nil,
        billing_thresholds_reset_billing_cycle_anchor: nil
      )
```